### DISCUSSION TITLE
Plugin in moderation state

### DISCUSSION THREAD

**Discussion by pardonmycode** [eclipse-oomph/oomph#88] [Q&A] [answered]
My Eclipse Plugin "Abap Action Chain" stay in moderation state over 2 Weeks? Is The Eclipse Marketing no more supported?

**Comment by merks** [comment]
I've opened this issue:

https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4634

I expect the answer should come from there.

**Reply by merks** [reply, marked as answer]
Please see the answer here:

https://gitlab.eclipse.org/eclipsefdn/it/websites/marketplace.eclipse.org/-/issues/234#note_2306686

Perhaps emails from the foundation are ending up in your spam folder?

Mail should come from there:

marketplace@eclipse-foundation.org

**Reply by pardonmycode** [reply]
Thanks, I already answered and linked the Eclipse plugin to a homepage. I didn't know that was necessary because it is an optional field. It should be marked as required.
